netci.groovy

```diff
@@ -19,6 +19,7 @@ def static getBuildJobName(def configuration, def os, def architecture) {
 platformList.each { platform ->
     // Calculate names
     def (os, architecture, configuration) = platform.tokenize(':')
+	def osDisplayName = os;
```

Comment:
Also, how does this fix it? Isn't osDisplayName always equals to OS? And you were already passing OS before.